grr/server/grr_response_server/gui/ui/components/client_details/entry_history_dialog/entry_history_dialog_test.ts

```diff
@@ -86,6 +87,7 @@ describe('Entry History Dialog', () => {
 
     expect(component.tableRows).toEqual(expectedTableRows);
 
+    // @ts-ignore
```

Comment:
Why do we need `@ts-ignore`?

Reply:
Because the property is `readonly`, changing it after it was assigned would signal an error, so I had to add `@ts-ignore` to bypass it. I thought about creating the whole object right there, but I would have to also configure the testing module. Do you think this would be a better approach, in order to inject the data?

Reply:
Please avoid disabling the error handling with @ts-ignore. This violates basic invariances (readonly attributes are never overwritten) and is unexpected for the person reading the code. Please refactor the test to create new objects and inject them.